src/DiskIO/DiskDaemon/DiskdAction.cc

```diff
@@ -65,13 +65,13 @@ DiskdAction::Create(const Mgr::CommandPointer &aCmd)
 DiskdAction::DiskdAction(const Mgr::CommandPointer &aCmd):
     Action(aCmd), data()
 {
-    debugs(79, 5, HERE);
+    debugs(79, 5, "");
```

Comment:
> If these debugs() lines are going to be touched they need fixing

I strongly disagree. Fixing object construction/destruction debugging is outside this PR scope. This PR automatically removes HERE. Nothing else is required.

If somebody comes up with a script that fixes more debugs() issues while removing HERE, then great! Otherwise, just removing HERE is sufficient.

Please keep in mind that combining any _manual_ changes with removing HERE is not a good idea because such an approach will make it impractical to replicate HERE-removal in other branches. For the overheads to approach bearable levels, global HERE removal must be automated.